cfme/tests/ansible/test_embedded_ansible_basic.py

```diff
@@ -253,6 +253,39 @@ def test_embedded_ansible_private_repository_crud(request, ansible_private_repos
     assert not ansible_private_repository.exists
 
 
+@pytest.mark.tier(2)
+@pytest.mark.meta(automates=[1845281])
+def test_embedded_ansible_file_type_repository_crud(appliance, wait_for_ansible, request):
+    """
+    Bugzilla:
+        1845281
+
+    Polarion:
+        assignee: gtalreja
+        casecomponent: Ansible
+        caseimportance: medium
+        initialEstimate: 1/6h
+    """
+    repo = cfme_data.ansible_links.playbook_repositories.embedded_ansible
+    dest = '/opt/ansible_playbooks'
+    appliance.ssh_client.run_command(f'git clone { repo } { dest }')
+    repositories = appliance.collections.ansible_repositories
+    repository = repositories.create(
+        name=fauxfactory.gen_alpha(),
+        url='file://' + f'{ dest }',
```

Comment:
This f-string should be combined, one of our code standards is never using `+` for text concatentation.

`url=f'file://{dest}'` is equivalent.

Reply:
Thanks @mshriver , updated the f-string.